book-example/src/format/theme/editor.md

```diff
@@ -0,0 +1,40 @@
+# Editor
+
+In addition to providing runnable code playpens, mdBook optionally allows them to be editable. In order to enable editable code blocks, the following needs to be added to the ***book.toml***:
+
+```toml
+[output.html.playpen]
+editable = true
+```
+
+To make a specific block available for editing, the attribute `editable` needs to be added to it:
+
+<pre><code class="language-markdown">```rust,editable
+fn main() {
+    let number = 5;
+    print!("{}", number);
+}
+```</code></pre>
+
+The above will result in this editable playpen:
+
+```rust,editable
+fn main() {
+    let number = 5;
+    print!("{}", number);
+}
+```
+
+Note the new `Undo Changes` button in the editable playpens.
+
+## Customizing the Editor
+
+By default, the editor is the [Ace](https://ace.c9.io/) editor, but, if desired, the functionality may be overriden by providing a different folder:
```

Comment:
It was originally intended that the playpen editor was also customizable in the same way the theme is: by providing a folder with files that are named the same. But I am a bit confused because the [theme](https://rust-lang-nursery.github.io/mdBook/format/theme/theme.html) page says that the new `theme` folder needs to be _next to_ `src`, while I always thought it was _inside_ `src`, and when I added a folder next to `src` rather than inside, mdBook did not report that it was copying anything.

I generally had trouble getting this functionality working regardless; I only ever recall theme/playpen replacement working if I manually specified the folder in the .toml. Not sure if bug, or I'm using it wrong, or... but since it's not functionality I can replicate I decided to not document it.

Reply:
I'm not exactly sure how it works either :( 

I'm planning to give the HTML renderer a massive makeover some time soon and in the process I'll be able to document how it actually does rendering. 

That kind of functionality should be easy to test. We just drop a `themes/` directory in the dummy book's `src/` (or next to it) and then make sure `mdbook` picks it up.